sdk/storage_blobs/src/container/operations/list_blobs.rs

```diff
@@ -23,6 +23,7 @@ operation! {
     ?include_deleted: bool,
     ?include_tags: bool,
     ?include_versions: bool,
+    ?marker: NextMarker,
```

Comment:
```suggestion
    ?initial_marker: NextMarker,
```

Will make the code lower easier to read I think, because 'marker' is contextless

Reply:
done